README.adoc

```diff
@@ -13,14 +13,19 @@ image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 :note-caption: :information_source:
 NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
 
-It is very easy to analyze a C, C++ and Objective-C project with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
+It is very easy to analyze a C, C++ and Objective-C project (and compute its test coverage) with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
```

Comment:
I would remove the parentheses because the main point of this example _is_ test coverage ^^
(the other changes are what Grammarly suggests...)
```suggestion
It is straightforward to analyze a C, C++, and Objective-C project and compute its test coverage with SonarCloud and https://docs.sonarcloud.io/getting-started/github/[GitHub Actions]:
```

Reply:
removed parens
I disagree with the grammarly here for the sake of consistency with other examples all of them use the same formula (and do not have the oxford comma either)